src/test/scala/edu/ie3/simona/test/common/AgentTypedSpec.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * © 2024. TU Dortmund University,
+ * Institute of Energy Systems, Energy Efficiency and Energy Economics,
+ * Research group Distribution grid planning and operation
+ */
+
+package edu.ie3.simona.test.common
+
+import com.typesafe.scalalogging.LazyLogging
+import org.apache.pekko.actor.ActorRef
+import org.apache.pekko.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import org.scalatest.PrivateMethodTester
+import org.scalatest.matchers.should
+import org.scalatest.wordspec.AnyWordSpecLike
+
+/** Class to help building tests for agents
+  */
+class AgentTypedSpec
```

Comment:
1. This is currently not used for testing agents
2. If it's a class and its name ends in `Spec`, scalatest thinks this should be executed
3. Do we need this at all? `ScalaTestWithActorTestKit with UnitSpec` should be enough for the vast majority of test suites